src/main/java/bio/terra/service/profile/ProfileService.java

```diff
@@ -237,11 +238,14 @@ public void createProfileIamResource(BillingProfileRequestModel request, Authent
 
     public void deleteProfileIamResource(String profileId, AuthenticatedUserRequest user) {
         // TODO: their may may not be a resource behind this profile. Therefore we
-        //  eat any not found exception called that a success. Move when we enable authorization.
+        //  eat any not found exception called that a success. Remove when we enable authorization.
```

Comment:
nit: it's not directly part of this pr, but I'm seeing 240 now and it looks odd

Reply:
Yeah, speech to text gets it wrong a lot.